context/brand-context/default/scss/00-tokens/_gap.variables.scss

```diff
@@ -1,6 +1,6 @@
 
 // Do not edit directly
-// Generated on Fri, 21 Oct 2022 11:11:43 GMT
+// Generated on Mon, 09 Jan 2023 13:28:25 GMT
```

Comment:
The aren't actually any changes to this file, I think we need a way to not generate this line when the file doesn't change as it's extremely noisy

Reply:
In fact is there any need for this "Generated on" message? Github shows the last changed date

Reply:
> I think we need a way to not generate this

Agreed, I have a task to look at seeing how we can 'test' what has changed and get Style Dictionary to act accordingly.

Reply:
> In fact is there any need for this "Generated on" message? 

I guess not - as the 'do not edit directly' message is there. 

> Github shows the last changed date

We're assuming folks would check this‽ Although tools like VSCode are quite adept at showing this information (even line-by-line).

Reply:
I think what we need is these files to be auto-generated as part of the package management, then there is no need fo anyone to check anything

Reply:
Yes, I'd love some form of automation for this so it's not "yet another `npm` script" that someone needs to run. It not being on [npmjs](http://npmjs.com) at the moment is probably scuppering possibilities. 

🤞🏼 the single release 🤞🏼